templates/templates/base.html

```diff
@@ -55,5 +55,6 @@
 
 <script src="/static/dist/global-nav.js"></script>
 
+
```

Comment:
Any reason for the extra blank line?
After such a valiant effort of removing other unnecessary lines in other files!

Reply:
I did what I could as I watched love island ... sorry >.<